pymemcache/client/hash.py

```diff
@@ -0,0 +1,282 @@
+import socket
+import time
+from pymemcache.client.base import Client, PooledClient, _check_key
+from clandestined import RendezvousHash as RH
+
+
+class RendezvousHash(RH):
+    def get_node(self, key):
+        return self.find_node(key)
+
+
+class HashClient(object):
+    """
+    A client for communicating with a cluster of memcached servers
+    """
+    def __init__(
+        self,
+        servers,
+        hasher=None,
+        serializer=None,
+        connect_timeout=None,
+        timeout=None,
+        no_delay=False,
+        socket_module=socket,
+        key_prefix=b'',
+        max_pool_size=None,
+        lock_generator=None,
+        retry_attempts=2,
+        retry_timeout=1,
+        dead_timeout=60,
+        use_pooling=False,
+    ):
+        """
+        Args:
+          servers: list(tuple(hostname, port))
+          serializer: optional class with ``serialize`` and ``deserialize``
+                      functions.
+          hasher: optional class three functions ``get_node``, ``add_node``,
+                  and ``remove_node``
+                  defaults to Rendezvous (HRW) hash.
+
+          use_pooling: use py:class:`.PooledClient` as the default underlying
+                       class. ``max_pool_size`` and ``lock_generator`` can
+                       be used with this. default: False
+
+          retry_attempts: Amount of times a client should be tried before it
+                          is marked dead and removed from the pool.
+          retry_timeout: Time in seconds that should pass between retry
```

Comment:
Can you mark this as "int" or "float", so it's more obvious whether fractional seconds are allowed. Same for the following.